wallet/chain/p/wallet/backend_visitor.go

```diff
@@ -120,10 +124,39 @@ func (b *backendVisitor) BaseTx(tx *txs.BaseTx) error {
 }
 
 func (b *backendVisitor) ConvertSubnetTx(tx *txs.ConvertSubnetTx) error {
+	for i, vdr := range tx.Validators {
+		b.b.setOwner(
+			tx.Subnet.Append(uint32(i)),
```

Comment:
> So this ID depends on the ordinal position of a given validator in `tx.Validators`?

Yes

> And that's safe to do because a given wallet instance is likely to result in the ordinal position changing?

This is safe to do because the validators in the transaction are ordered (in the slice). This order is canonical (and can not be modified). Additionally, there is only one possible `ConvertSubnetTx` that will be allowed to be accepted per `subnetID`.